lib/cryptoexchange/exchanges/codex/services/market.rb

```diff
@@ -0,0 +1,48 @@
+module Cryptoexchange::Exchanges
+  module Codex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Codex::Market::API_URL}/tickers"
+        end
+
+        def adapt_all(output)
+          output['data'].map do |key, pair|
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+                base:   pair['base_unit'],
+                target: pair['quote_unit'],
+                market: Codex::Market::NAME
+            )
+
+            adapt(market_pair, pair)
```

Comment:
`adapt(market_pair, pair)` inconsistent with the argument name below, causes confusion

Reply:
done